fix(stdlib): doc comments and parameter names (#2739)
tact-lang/tact: docs/src/content/docs/ref/core-send.mdx

<Badge text="Gas-expensive" title="Uses 500 gas units or more" variant="danger" size="medium"/><p/>

```tact
fun nativeSendMessage(msg: Cell, mode: Int);
```

> [Queues the `msg` message](/book/send#outbound-message-processing) to be sent by specifying the complete message cell.

Reviewer: Hmm, why this change? We don't give raw links in the docs whenever possible

Author: we just should generate the reference docs from the source code, it's just mind-numbing to do the same work twice

> [Queues the `msg` message](/book/send#outbound-message-processing) to be sent by specifying the complete message cell.
> The `mode` parameter configures how to send a message, see: https://docs.tact-lang.org/book/message-mode.

Reviewer: ```suggestion
[Queues the message](/book/send#outbound-message-processing) to be sent by specifying the complete `msg` cell and the [message `mode`](/book/message-mode).
```

Author: nope, I'm not doing it here, once @i582 implements the generator, this will go away

Reviewer: > nope, I'm not doing it here, once i582 implements the generator, this will go away

But I will. Also, we'll need more talks regarding generator to preserve:
1. links in the notes
2. other stuff (will discuss later anyways, this msg would instantly get lost)
